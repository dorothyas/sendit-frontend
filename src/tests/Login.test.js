import React from 'react';
import renderer from 'react-test-renderer';
import Login from '../components/Login';

describe('Login Page', () => {
    it('should render without crashing', () => {
        const wrapper = renderer.create(<Login />);
        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});
