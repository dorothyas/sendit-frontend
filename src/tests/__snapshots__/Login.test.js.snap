// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Page should render without crashing 1`] = `
<h1>
  Login
</h1>
`;
